modules/app.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable no-plusplus */
 /* eslint-disable radix */
+import navigators from '../nav.js'
```

Comment:
- [ ] I think it will be better if you move your `nav,js` file which contains the navbar component in the modules folder. This is because as in the requirements, you are expected to divide the features and functionalities into simple files called modules and put them in the modules folder altogether, This one is the best example. That is why I think it has to be in the modules folder 😺